docs/index.rst

```diff
@@ -50,36 +50,31 @@ introduce concepts that build upon one another.
 ============================
 Latest news
 ============================
-----------------------------
-Changes to Inheritance Tools
-----------------------------
 
-As of version 0.16.0, the built-in Mendelian inheritance tools are more stringent by default (they can be relaxed with the ``--lenient``) option. By default, samples with unknown phenotype will not affect the tools, and
-strict requirements are placed on family structure. See the the :doc:`docs <content/tools>` for
-more info. In addition, the inheritance tools now support multi-generational pedigrees.
+----------------
+New Installation 
+----------------
 
+In version 0.18, we have introduced a new installation procedure based on `conda <http://conda.pydata.org/docs/>`_ that
+should make the installation more reliable.
+For users with an existing installation with any trouble using `gemini update --devel`,
+we suggest to do a fresh install using a command like:
 
-----------------------------
-Much faster genotype queries
-----------------------------
-As of version 0.15.0, GEMINI creates auxilliary index files
-using the `bcolz` library. This allows queries that filter
-on sample genotype information via the ``--gt-filter`` option
-to be **up to 1000 times faster than identical queries using
-version 0.14.0 or earlier**. Details of the implementation and caveats can be found :doc:`here <content/genotype_query_engines>`.
+.. code-block::
 
-.. note::
+    wget https://github.com/arq5x/gemini/raw/master/gemini/scripts/gemini_install.py
+    python gemini_install.py $tools $data
+    PATH=$tools/bin:$data/anaconda/bin:$PATH
 
-    In order to expedite your queries with the new bcolz indexing strategy, one must invoke the ``--use-bcolz``
-    option. We will likely phase this option out over time.
-    For example::
+where `$tools` and `$data` are paths writable on your system.
 
```

Comment:
We could also add details to re-install in place to avoid folks having to re-download all of the data. Something like,

With an existing $tool and $data directory from a previous install, you can use the installer to re-install the Python code with the new version, but leave the existing data in place. To do this, first remove the old anaconda directory:
```
rm -rf $data/anaconda
```
then run the installation commands above.